<launch>
  <arg name="results_path" default="/home/helen/data/jfr_2018/local_benchmark/local_24_10_2018_continue.csv" />

  <arg name="frame_id" default="odom" />

  <node name="local_planning_benchmark" pkg="mav_planning_benchmark" type="local_planning_benchmark_node" output="screen" args="--alsologtostderr">
    <param name="results_path" value="$(arg results_path)" />

    <param name="visualize" value="false" />
    <param name="robot_radius" value="0.5" />
    <param name="tsdf_voxel_size" value="0.20" />
    <param name="tsdf_voxels_per_side" value="16" />
    <param name="clear_sphere_radius" value="1.5" />
    <param name="occupied_sphere_radius" value="4.0" />
    <param name="update_mesh_every_n_sec" value="0.0" />
    <param name="num_seconds_to_plan" value="5.0" />
    <param name="camera_max_dist" value="20.0" />
    <param name="max_ray_length_m" value="5.0" />
    <param name="frame_id" value="$(arg frame_id)" />
    <param name="world_frame" value="$(arg frame_id)" />
    <param name="publish_pointclouds" value="true" />
    <param name="verbose" value="true" />

  </node>

</launch>
